.github/workflows/stable-compilation.yml

```diff
@@ -0,0 +1,59 @@
+name: Stable Compilation
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  docker:
+    name: ${{ matrix.image }}
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.image }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - ubuntu:18.04  # CMake 3.10.2 - g++ 7.4.0  - LTS
+          - debian:10     # CMake 3.13.4 - g++ 8.3.0  - old
+          - ubuntu:20.04  # CMake 3.16.3 - g++ 9.3.0  - LTS
+          - ubuntu:21.04  # CMake 3.18.4 - g++ 10.3.0 - stable
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.8.0
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          export DEBIAN_FRONTEND="noninteractive"
+          apt-get update
+          apt-get install -y --no-install-recommends --no-install-suggests \
+            build-essential cmake ninja-build git \
+            libicu-dev libexpat1-dev
```

Comment:
This is also great for an updated "how to install" guide because I see the package names to install :D

Reply:
Debian still has funny names for some libs. For player it is worse.